pkg/controllers/scheduler/schedulingtriggers.go

```diff
@@ -276,3 +276,55 @@ func (s *Scheduler) enqueueFederatedObjectsForCluster(cluster pkgruntime.Object)
 		s.worker.EnqueueObject(fedObject)
 	}
 }
+
+// enqueueFederatedObjectsForProfile enqueues all federated objects that match the profile
+func (s *Scheduler) enqueueFederatedObjectsForProfile(profile pkgruntime.Object) {
```

Comment:
Not a request for changes, just a general comment:
This looks very dangerous, do does `enqueueFederatedObjectsForPolicy`. We may need to rethink how we response to policy and profile events.

Reply:
we can create an issue to discuss this